tspec/base/c/c89/errno.h.ts

```diff
@@ -4,7 +4,10 @@
 # See doc/copyright/ for the full copyright terms.
 
 # 4.1.3 Errors <errno.h>
-+EXP lvalue int errno;
+
++FUNC (extern) int *__errno_location(void);
++DEFINE errno %% (*__errno_location ()) %%;
+
```

Comment:
Hm. I think this is the wrong place to do this. The tspec file here is supposed to convey what the C89 standard says. Which is:
```
   The macros are 

         EDOM
         ERANGE

which expand to distinct nonzero integral constant expressions; and 

         errno

which expands to a modifiable lvalue/83/ that has type int , the value
of which is set to a positive error number by several library
functions.  It is unspecified whether errno is a macro or an
identifier declared with external linkage.  If a macro definition is
suppressed in order to access an actual object, or a program defines
an external identifier with the name errno , the behavior is
undefined.
```

(also changing this API definition would of course break for other OSes)

So however we deal with this (I'm not sure how exactly), I think the "dealing with" part has to live in the OS-dependent glue, somewhere under /osdep

Reply:
I'll take a look over there